src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -35,7 +37,7 @@
  * <code>`a.getId().equalsIgnoreCase(b.getId())`</code> are true then all must
  * be true.</p>
  */
-public interface CatalogType {
+public interface CatalogType extends TextRepresentable {
```

Comment:
PS in the current text.of implemention you can indeed change the display style/format of a catalog type. `Text.of(RED, EntityTypes.ZOMBIE)`